src/components/TeamMember.js

```diff
@@ -0,0 +1,38 @@
+import React from 'react';
+import 'aos/dist/aos.css';
+import '././TeamMember/TeamMember.css';
+
+export default function TeamMember({ isRight, name, image }) {
+  // const isRight = props.isRight;
+  if (isRight)
+    return (
+      <>
+        <div className=" h-64 w-full bg-gray-900 flex overflow-hidden my-10">
```

Comment:
can you reduce the width of the image? Cause we can put images that wide 
lets make the card look like this
![image](https://user-images.githubusercontent.com/12795240/89955759-63477f80-dc3c-11ea-8297-aaffbeb53ffa.png)